source/playbooks/customize-a-playbook.rst

```diff
@@ -35,7 +35,8 @@ You can change a playbook’s configuration at any time, but changes will only b
    - To edit the playbook directly select the actions menu next to the playbook name, then select **Edit**.
    - To access the playbook dashboard, select the hyperlinked playbook name.
 
-4. Configure the playbook the same way a playbook is created or edited.
+4. Select the **Outline** tab. 
+5. Configure the playbook the same way a playbook is created or edited.
```

Comment:
> "... or edited."

This is slightly confusing since this makes me assume that there's another section that explains how a playbook is edited, whereas this is the edit playbook section of the docs. Not sure if I'm interpreting this incorrectly though.

Reply:
Great catch. I'll revise.